src/__tests__/__mocks__/data.js

```diff
@@ -0,0 +1,1767 @@
+    today_new_open_cases: 0,
+    today_new_recovered: 0,
+    today_open_cases: 350967,
+    today_recovered: 1086913,
+    today_vs_yesterday_confirmed: 0,
+    today_vs_yesterday_deaths: 0,
+    today_vs_yesterday_open_cases: 0,
+    today_vs_yesterday_recovered: 0,
+    yesterday_confirmed: 1458544,
+    yesterday_deaths: 20664,
+    yesterday_open_cases: 350967,
+    yesterday_recovered: 1086913,
+  },
+  {
+    date: '2022-02-03',
+    id: 'bahia',
+    links: [
+      {
+        href: '/api/2022-02-03/country/brazil/region/bahia',
+        rel: 'self',
+        type: 'GET',
+      },
+    ],
+    name: 'Bahia',
+    name_es: 'Bahia',
+    name_it: 'Bahia',
+    source: 'Ministério da Saúde Brasil',
+    sub_regions: [],
+    today_confirmed: 1374028,
+    today_deaths: 28026,
+    today_new_confirmed: 0,
+    today_new_deaths: 0,
+    today_new_open_cases: 0,
+    today_new_recovered: 0,
+    today_open_cases: 178531,
+    today_recovered: 1167471,
+    today_vs_yesterday_confirmed: 0,
+    today_vs_yesterday_deaths: 0,
+    today_vs_yesterday_open_cases: 0,
+    today_vs_yesterday_recovered: 0,
+    yesterday_confirmed: 1374028,
+    yesterday_deaths: 28026,
+    yesterday_open_cases: 178531,
+    yesterday_recovered: 1167471,
+  },
+  {
+    date: '2022-02-03',
+    id: 'sergipe',
+    links: [
+      {
+        href: '/api/2022-02-03/country/brazil/region/sergipe',
+        rel: 'self',
+        type: 'GET',
+      },
+    ],
+    name: 'Sergipe',
+    name_es: 'Sergipe',
+    name_it: 'Sergipe',
+    source: 'Ministério da Saúde Brasil',
+    sub_regions: [],
+    today_confirmed: 297332,
+    today_deaths: 6109,
+    today_new_confirmed: 0,
+    today_new_deaths: 0,
+    today_new_open_cases: 0,
+    today_new_recovered: 0,
+    today_open_cases: 26724,
+    today_recovered: 264499,
+    today_vs_yesterday_confirmed: 0,
+    today_vs_yesterday_deaths: 0,
+    today_vs_yesterday_open_cases: 0,
+    today_vs_yesterday_recovered: 0,
+    yesterday_confirmed: 297332,
+    yesterday_deaths: 6109,
+    yesterday_open_cases: 26724,
+    yesterday_recovered: 264499,
+  },
+  {
+    date: '2022-02-03',
+    id: 'rondonia',
+    links: [
+      {
+        href: '/api/2022-02-03/country/brazil/region/rondonia',
+        rel: 'self',
+        type: 'GET',
+      },
+    ],
+    name: 'Rondonia',
+    name_es: 'Rondonia',
+    name_it: 'Rondonia',
+    source: 'Ministério da Saúde Brasil',
+    sub_regions: [],
+    today_confirmed: 323699,
+    today_deaths: 6844,
+    today_new_confirmed: 0,
+    today_new_deaths: 0,
+    today_new_open_cases: 0,
+    today_new_recovered: 0,
+    today_open_cases: 66600,
+    today_recovered: 250255,
+    today_vs_yesterday_confirmed: 0,
+    today_vs_yesterday_deaths: 0,
+    today_vs_yesterday_open_cases: 0,
+    today_vs_yesterday_recovered: 0,
+    yesterday_confirmed: 323699,
+    yesterday_deaths: 6844,
+    yesterday_open_cases: 66600,
+    yesterday_recovered: 250255,
+  },
+];
+
+export default data;
```

Comment:
- [ ] One of your tests is failing. As indicated by the message, I don't think `data.js` is a test file but it is included when running tests. Kindly fix this.